<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BillDialog</class>
 <widget class="QDialog" name="BillDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>815</width>
    <height>400</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>justagain DIN</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>اضافة فاتورة</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/icons/mainicon.ico</normaloff>:/icons/mainicon.ico</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>795</width>
       <height>334</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_13">
      <item row="2" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>قراءة</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>81</width>
            <height>31</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>اضافة</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="minimumSize">
           <size>
            <width>81</width>
            <height>31</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>الغاء</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QGridLayout" name="gridLayout_14">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_12">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>161</width>
            <height>71</height>
           </size>
          </property>
          <property name="title">
           <string>تاريخ الفاتوره</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_12">
           <item row="0" column="0">
            <widget class="QDateEdit" name="dateEdit_2">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="wrapping">
              <bool>false</bool>
             </property>
             <property name="displayFormat">
              <string>yyyy-MM-dd</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_11">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>181</width>
            <height>71</height>
           </size>
          </property>
          <property name="title">
           <string>رقم الفاتوره</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="0" column="0">
            <widget class="QLineEdit" name="bill_number">
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="groupBox_10">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>271</width>
            <height>71</height>
           </size>
          </property>
          <property name="title">
           <string>اسم المورد</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="0">
            <widget class="QLineEdit" name="supplier">
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>196</width>
            <height>117</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="title">
           <string>المنتج</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QTabWidget" name="tabWidget">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="elideMode">
              <enum>Qt::ElideNone</enum>
             </property>
             <property name="documentMode">
              <bool>false</bool>
             </property>
             <property name="tabsClosable">
              <bool>false</bool>
             </property>
             <property name="movable">
              <bool>false</bool>
             </property>
             <widget class="QWidget" name="tab">
              <attribute name="icon">
               <iconset>
                <normalon>:/icons/Search-good-icon.png</normalon>
               </iconset>
              </attribute>
              <attribute name="title">
               <string>الرقم</string>
              </attribute>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLineEdit" name="Numberinput">
                 <property name="maxLength">
                  <number>999999999</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_2">
              <attribute name="icon">
               <iconset>
                <normalon>:/icons/Monitoring-icon.png</normalon>
               </iconset>
              </attribute>
              <attribute name="title">
               <string>بحث</string>
              </attribute>
              <layout class="QFormLayout" name="formLayout_2">
               <item row="0" column="0">
                <widget class="QLineEdit" name="search_item">
                 <property name="focusPolicy">
                  <enum>Qt::StrongFocus</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="minimumSize">
           <size>
            <width>141</width>
            <height>111</height>
           </size>
          </property>
          <property name="title">
           <string>الكميه الحاليه</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLineEdit" name="current_amount">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="minimumSize">
           <size>
            <width>151</width>
            <height>111</height>
           </size>
          </property>
          <property name="title">
           <string>الكميه المباعه</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLineEdit" name="sold_amount">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="minimumSize">
           <size>
            <width>151</width>
            <height>111</height>
           </size>
          </property>
          <property name="title">
           <string>الكميه الإجماليه</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLineEdit" name="total_amount">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_15">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>121</width>
            <height>101</height>
           </size>
          </property>
          <property name="title">
           <string>الكميه الجديده</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLineEdit" name="added_amount">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QGroupBox" name="groupBox_8">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>151</width>
            <height>101</height>
           </size>
          </property>
          <property name="title">
           <string>سعر الجمله</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QLineEdit" name="wholesale_price">
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="groupBox_7">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>141</width>
            <height>101</height>
           </size>
          </property>
          <property name="title">
           <string>سعر البيع</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QLineEdit" name="sell_price">
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QGroupBox" name="groupBox_6">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>141</width>
            <height>101</height>
           </size>
          </property>
          <property name="title">
           <string>الصلاحيه</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QDateEdit" name="dateEdit">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="wrapping">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="displayFormat">
              <string>yyyy-MM-dd</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_9">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>141</width>
            <height>101</height>
           </size>
          </property>
          <property name="title">
           <string>سعر الشراء</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0">
            <widget class="QLineEdit" name="buy_price">
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>Numberinput</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>added_amount</tabstop>
  <tabstop>sell_price</tabstop>
  <tabstop>wholesale_price</tabstop>
  <tabstop>buy_price</tabstop>
  <tabstop>bill_number</tabstop>
  <tabstop>supplier</tabstop>
  <tabstop>total_amount</tabstop>
  <tabstop>current_amount</tabstop>
  <tabstop>sold_amount</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>search_item</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>dateEdit_2</tabstop>
 </tabstops>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>BillDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>794</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>716</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sell_price</sender>
   <signal>returnPressed()</signal>
   <receiver>wholesale_price</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bill_number</sender>
   <signal>returnPressed()</signal>
   <receiver>supplier</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>528</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>dateEdit_2</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>695</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateEdit_2</sender>
   <signal>editingFinished()</signal>
   <receiver>bill_number</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>695</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>528</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>supplier</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>794</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>added_amount</sender>
   <signal>returnPressed()</signal>
   <receiver>buy_price</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>725</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>616</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buy_price</sender>
   <signal>returnPressed()</signal>
   <receiver>sell_price</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wholesale_price</sender>
   <signal>returnPressed()</signal>
   <receiver>dateEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
